src/interface/src/app/map/map.component.ts

```diff
@@ -340,7 +340,7 @@ export class MapComponent implements AfterViewInit, OnDestroy, OnInit, DoCheck {
 
     // Renders the selected region on the map.
     this.selectedRegion$
-      .pipe(untilDestroyed(this))
```

Comment:
this subscription was leaking as we call this each time we init a map. Using take(1) instead to end subscription right away